docs/0-Preparing/starting_side_projects.md

```diff
@@ -0,0 +1,93 @@
+
+A side project can loosely be defined as "code that you write outside of an obligation". They're great for learning and employers like to see them on resumes. Anyone can create side projects, regardless of your experience level!
+
+Nor do these have to be big or take too long to do! It's totally okay to make something assignment sized that takes less than a week of work, or a few hours on a weekend.
+
+Hopefully, this section will provide you some insights on how to think about side projects and ideas on how to get started.
+
+### Benefits
+
+- It's a chance everyone has to write code. Getting better at coding requires writing code!
+- It's a chance to build something from scratch
+    - You don't always get to do that at work
+    - You get to see the whole picture
+    - It's satisfying
+- You can learn and explore different technologies with little commitment
+- They can be fun and allow you to enjoy programming in a way that your job can't
+- They help get you employed
+- They really help get you employed when you don't have industry experience
+- They open up more doors even when you have experience
+
+### Important notice
+
+While side projects have a lot of benefits, it doesn't mean that they should take up all your free time. **PLEASE PLEASE please don't spend all your time on coding projects unless that's just how you like to spend your time** you don't have to do side projects to be successful even though it often is talked about as if you do.
+
+### Side projects from the perspective of the employer
+
+Among other misconceptions, side projects don't have to be novel, amazing, or even useful. For internships and junior positions especially, all that an employer cares about is *interest and willingness to learn*. Here are different "signals" that a side project can convey, sorted roughly by the amount of experience associated with them. **Any one of these can be interesting for an employer**!
+
+**You can learn a technology on your own**
+
+- *Example*: You're reading through a book on learning language X, and applying what you learned by coding a few Project Euler problems.
+- *Why it's useful*: Most employers will hire you even if you don't know all the technologies that they use, and will expect you to learn some on the job. Here, you've established that you can do that.
+
+**You can learn a concept on your own**
+
+- *Example*: You got interested in Big Data and read some blog posts about how work can be split and parallelized to run on multiple machines.
+- *Why it's useful*: Employers sometimes have hard problems to solve and like to know that they can give you a bunch of readings on a new concept.
+
+**You can come up with project ideas**
+
+- *Example*: You noticed that the school newspaper club manually tallies up survey results into different demographics. You write a script that automates this work for them and generates pretty graphs.
+- *Why it's useful*: Employers like to see that you can take initiative.
```

Comment:
random nit: I noticed you have periods at the end in this file but not the similar place in the other file